crates/wit-parser/src/resolve.rs

```diff
@@ -1329,9 +1329,22 @@ impl Resolve {
         }
     }
 
-    fn include_stability(&self, stability: &Stability) -> bool {
+    fn include_stability(&self, stability: &Stability, pkg_id: Option<&PackageId>) -> bool {
```

Comment:
It's ok to take a `PackageId` argument here directly since the optional cases below using `as_ref` should all be filled in earlier during resolve so I believe it should be safe to `.unwrap()`

Reply:
This required more changes than expected -- signatures of some of the update fns needed to change